apps/E2E/src/ContextualMenu/specs/ContextualMenu.spec.win.ts

```diff
@@ -12,41 +12,45 @@ describe('ContextualMenu Testing Initialization', function () {
   it('Click and navigate to ContextualMenu test page', async () => {
     /* Click on component button to navigate to test page */
     await NavigateAppPage.clickAndGoToContextualMenuPage();
-    await ContextualMenuPageObjectObject.waitForPageDisplayed(PAGE_TIMEOUT);
+    await ContextualMenuPageObject.waitForPageDisplayed(PAGE_TIMEOUT);
 
-    await expect(await ContextualMenuPageObjectObject.isPageLoaded()).toBeTruthy(ContextualMenuPageObjectObject.ERRORMESSAGE_PAGELOAD);
-    await expect(await ContextualMenuPageObjectObject.didAssertPopup()).toBeFalsy(ContextualMenuPageObjectObject.ERRORMESSAGE_ASSERT); // Ensure no asserts popped up
+    await expect(await ContextualMenuPageObject.isPageLoaded()).toBeTruthy(ContextualMenuPageObject.ERRORMESSAGE_PAGELOAD);
+    await expect(await ContextualMenuPageObject.didAssertPopup()).toBeFalsy(ContextualMenuPageObject.ERRORMESSAGE_ASSERT); // Ensure no asserts popped up
   });
 });
 
 describe('ContextualMenu Functional Tests', async () => {
   /* Scrolls and waits for the ContextualMenu to be visible on the Test Page */
   beforeEach(async () => {
-    await ContextualMenuPageObjectObject.scrollToTestElement();
+    await ContextualMenuPageObject.scrollToTestElement(await ContextualMenuPageObject._contextualMenu);
 
-    await ContextualMenuPageObjectObject.sendKey(ContextualMenuSelector.ContextualMenu, Keys.ESCAPE); // Reset ContextualMenu state for next test
+    await ContextualMenuPageObject.closeContextualMenu(); // Reset ContextualMenu state for next test
   });
 
-  it('Click on ContextualMenu Button and validate that the list of ContextualMenu Items open', async () => {
+  it('Click on ContextualMenu Button -> Validate that the menu opens by checking if its items are displayed', async () => {
     /* Click on the ContextualMenu */
-    await ContextualMenuPageObjectObject.clickComponent();
-    await ContextualMenuPageObjectObject.waitForContextualMenuItemsToOpen(PAGE_TIMEOUT);
+    await ContextualMenuPageObject.click(ContextualMenuPageObject._contextualMenu);
 
-    await expect(await ContextualMenuPageObjectObject.contextualMenuItemDisplayed()).toBeTruthy();
-    await expect(await ContextualMenuPageObjectObject.didAssertPopup()).toBeFalsy(ContextualMenuPageObjectObject.ERRORMESSAGE_ASSERT); // Ensure no asserts popped up
+    await expect(await ContextualMenuPageObject.waitForContextualMenuItemsToOpen(PAGE_TIMEOUT)).toBeTruthy(
+      'Clicked the ContextualMenu button, but the ContextualMenu items failed to open and display.',
```

Comment:
I don't think it's necessary to provide an error message here. I think if you change the error message in `waitForContextualMenuItemsToOpen/Display` to "Contextual Menu Items did not display." that should be enough. Same for the next two tests. We'll have the context of what action is preformed from the name of the test.